SolutionInfo.cs

```diff
@@ -1,11 +1,12 @@
 ﻿// <auto-generated/>
+
 using System.Reflection;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyProductAttribute("Octokit")]
-[assembly: AssemblyVersionAttribute("0.14.0")]
-[assembly: AssemblyFileVersionAttribute("0.14.0")]
-[assembly: ComVisibleAttribute(false)]
+[assembly: AssemblyProduct("Octokit")]
```

Comment:
I wonder if this will be reverted the next time I run it through the FAKE build script...